test/checkPages_test.js

```diff
@@ -50,7 +50,7 @@ function runTest(options, callback) {
 function testOutput(test, log, error, exception) {
   return function(err, context, count) {
     if (err || exception) {
-      test.equal(err.message, exception, 'Wrong exception text');
```

Comment:
This was kind of a deliberate attempt to ensure that err always had a .message property (i.e., was an Error). Do you recall the scenario that required you to make a change here?